fe/fe-core/src/main/java/com/starrocks/load/DeleteHandler.java

```diff
@@ -470,6 +499,18 @@ public boolean cancelJob(DeleteJob job, CancelType cancelType, String reason) {
         }
         LOG.info("start to cancel delete job, transactionId: {}, cancelType: {}", job.getTransactionId(),
                 cancelType.name());
+
+        // create push task for each backends
+        List<Long> backendIds = Catalog.getCurrentSystemInfo().getBackendIds(true);
+        for (Long backendId : backendIds) {
+            PushTask pushTask = new PushTask(backendId, TPushType.CANCEL_DELETE, TPriority.HIGH,
+                    TTaskType.REALTIME_PUSH, job.getTransactionId(),
+                    Catalog.getCurrentGlobalTransactionMgr().getTransactionIDGenerator().getNextTransactionId());
+            AgentTaskQueue.removePushTaskByTransactionId(backendId, job.getTransactionId(),
+                    TPushType.DELETE, TTaskType.REALTIME_PUSH);
+            AgentTaskQueue.addTask(pushTask);
```

Comment:
1. If the cancel delete task fails, when will it be cleaned up from the `AgentTaskQueue`? I think it is unnecessary to add the task to queue, and `AgentTaskExecutor` submits the task is enough.
2. Better to send the cancel delete tasks to the backends with delete tasks.